code/modules/projectiles/guns/energy.dm

```diff
@@ -160,6 +160,7 @@
 		cell.use(shot.e_cost)//... drain the cell cell
 	chambered = null //either way, released the prepared shot
 	recharge_newshot() //try to charge a new shot
+	return ..()
```

Comment:
It's pretty inconsistent to do `return ..()` here but in another process_chamber override, you do `. = ..()`. It could lead to some complicated bugs when registering signals.